### PR TITLE
added option to select font size

### PR DESCRIPTION
**Pull request by alexban011**
I really hate when companies make you restart the game to apply a setting just because they don't want to spend that much time making the setting change without restart...

..which is why I hate myself for this but I really tried making it change on runtime but just couldn't. Everything I tried either only updated parts of the UI or most of it but not all of it...

The setting goes from 0.8 to 1.5

These screenshots are with the setting on 1.5 (max), which would be a very rare case really, and resolution is 900x600. Increasing the resolution keeps the text big and looks much better, but I thought I should send the (almost) worst case scenario

fontSize1
fontSize1portrait
fontSize2
fontSize2Portrait

### CODE REVIEW

**Comment by alexban011** [comment]
> The font-factory and skin reloads

I tried those but they didn't scale the text right. Let me try again with those

**Review by yairm210 — APPROVED** [review]
Looks fine to me

File: core/src/com/unciv/ui/options/AdvancedTab.kt [hunk 1]
```diff
@@ -131,6 +132,31 @@ fun addFontFamilySelect(table: Table, settings: GameSettings, selectBoxMinWidth:
...
 
+private fun addFontSizeMultiplier(
+    table: Table,
+    settings: GameSettings,
+    selectBoxMinWidth: Float,
+    onFontChange: () -> Unit
+) {
+    table.add("Font size multiplier".toLabel()).left().fillX()
+
+    val fontSelectBox = SelectBox<Float>(table.skin)
```
[the remark this answers is not in the document]
Comment (alexban011) [reply core/src/com/unciv/ui/options/AdvancedTab.kt:143]: thats what I originally used while testing but it kept updating the screen every time I touched the slider and looked weird. Is there a way to only update after the user "let go" of the slider?

[the remark this answers is not in the document]
Comment (alexban011) [reply core/src/com/unciv/ui/options/AdvancedTab.kt]: Thanks to both of you :D 
I replaced it with a slider and also lowered the min value to 0.7 as that still looked decent
